### PR TITLE
Ece651 28 fe re design categories list

### PR DESCRIPTION
**Pull request by quanquach512**

### CODE REVIEW

**Comment by quanquach512** [comment]
Screenshot 2023-02-20 at 10 35 19 PM
Screenshot 2023-02-20 at 10 35 25 PM

File: src/components/SearchItemList.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
 import React from "react";
+import Header from './Header';
 
 const SearchItemList = props => {
-    return <div>Search items page.</div>
+    return (
+        <>
+            <Header />
+            {/*your code goes from here */}
```
Comment (quanquach512) [inline src/components/SearchItemList.js:8]: @DelinM  you can update your component below my comment to see if it works

Comment (DelinM) [reply src/components/SearchItemList.js:8]: Sounds good, we can have this pull request merged, and I will rebase my branch based on it.

**Comment by quanquach512** [comment]
@DelinM @Yaoxuannn solved PR conflicts

**Pull request merged**